test/conformance/ingress/run.go

```diff
@@ -68,5 +68,6 @@ func RunConformance(t *testing.T) {
 	if test.ServingFlags.EnableAlphaFeatures {
 		// Add your conformance test for alpha features
 		t.Run("headers/tags", TestTagHeaders)
+		t.Run("rewrite", TestRewriteHost)
```

Comment:
```suggestion
		t.Run("headers/host-rewrite", TestRewriteHost)
```

Reply:
not sure about `headers/`, this feature is pretty separate from the header-based routing feature, I think? Updated to `host-rewrite` though.

Reply:
My thinking was this was manipulating the host header prior to routing. 

`host-rewrite` works for me.